15/umbraco-cms/fundamentals/setup/upgrading/version-specific/README.md

```diff
@@ -29,8 +29,25 @@ Snapshots have been removed, meaning any code using `IPublishedSnapshot`, and by
 
 Models generated by ModelsBuilder used the `IPublishedSnapshot` interface, which has been removed. This means that the models need to be rebuilt. The approach to this will differ depending on the mode chosen:
 
-* InMemoryAuto - Remove the `umbraco\Data\TEMP\InMemoryAuto` folder to trigger a rebuild of the models.
-* SourceCodeAuto and SourceCodeManual - Remove the old models located in the `\umbraco\models` folder by default. Trigger a rebuild of the models from the ModelsBuilder dashboard in the backoffice.
+**InMemoryAuto**
+
+Remove the `umbraco\Data\TEMP\InMemoryAuto` folder to trigger a rebuild of the models.
+
+**SourceCodeAuto and SourceCodeManual**
+
+Remove the old models located in the `\umbraco\models` folder by default. This will cause your views to no longer be able to build due to missing types.
+To get around this you can disable precompiled view temporarily by adding the following to your csproj file:
+
+```xml
+<PropertyGroup>
+  <RazorCompileOnBuild>false</RazorCompileOnBuild>
+  <RazorCompileOnPublish>false</RazorCompileOnPublish>
+</PropertyGroup>
+```
+This will allow your site to start up, but you will still see an error page when trying to load a page. Disregard the error, enter the backoffice, and rebuild the models from the ModelsBuilder dashboard. You can now re-enable precompiled views and rebuild your site.
```

Comment:
```suggestion

This will allow your site to start up, but you will still see an error page when loading a page. 

1. Disregard the error.
2. Enter the backoffice.
3. Rebuild the models from the ModelsBuilder dashboard.

You can now re-enable precompiled views and rebuild your site.
```